tests/test_moscot_interface.py

```diff
@@ -0,0 +1,178 @@
+import pytest
+
+import scanpy as sc
+import cellrank.external as cre
+from anndata import AnnData
+from cellrank.kernels import MoscotKernel, ConnectivityKernel
+from cellrank.external.kernels._utils import MarkerGenes
+from cellrank.kernels._transport_map_kernel import SelfTransitions
+
+import numpy as np
+import pandas as pd
+from scipy.sparse import issparse, csr_matrix
+from pandas.core.dtypes.common import is_categorical_dtype
+
+import matplotlib.pyplot as plt
+from matplotlib.cm import get_cmap
+from matplotlib.colors import to_hex
+
+
+def _moscot_not_installed() -> bool:
+    try:
+        import moscot
+
+        return False
+    except ImportError:
+        return True
+
+
+moscot_not_installed_skip = pytest.mark.skipif(
+    _moscot_not_installed(), reason="moscot is not installed."
+)
+
+
+@moscot_not_installed_skip
```

Comment:
You can skip the whole file using `pytest` without the need to mark it like this.